docs/settings.md

```diff
@@ -299,7 +275,31 @@ Content-type: application/json
 To use this endpoint, an application needs a permission on the type
 `io.cozy.settings` for the verb `PUT`.
 
-### PUT /settings/instance/confirm_mail_tfa
+### PUT /settings/instance/activate_tfa
```

Comment:
Maybe we can use a more resourceful URLs like `PUT /settings/instance/tfa` and `POST /settings/instance/tfa/code` (I don't like verbs in paths, it's too RPC in my taste)

Reply:
And later, we may have `DELETE /settings/instance/tfa` to disable two-factor authentication